src/components/pages/geographyOriginalPage.tsx

```diff
@@ -217,6 +221,48 @@ export const GeographyOriginalPage = ({ geography, summary, targets, theme, them
     }
   };
 
+  /** Vespa search results */
+  const vespaSearchTabbedNavItems = themeConfig.categories
+    ? themeConfig.categories.options.map((category) => {
+        return {
+          title: category.label,
+          /** We need to maintain the slug to to know what to send to Vespa for querying. */
+          slug: category.slug,
+          // TODO: Make this work
+          count: 0,
```

Comment:
There is an attribute on the search result object called `"total_family_hits"` that I believe we should be able to use for this number?

Only issue is retrieving the number _before_ we select the category though 🤔

Reply:
Yea, I've left this as a "V2" to come up with a solution that works out all counts at once.